lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/NetworkSpec.hs

```diff
@@ -305,9 +330,10 @@ mockNetworkLayer
     -> StateT S m (TestNetworkLayer m)
 mockNetworkLayer logLine = do
     let jm = mockJormungandrClient logLine
+    Quantity k <- gets mockNodeK
```

Comment:
With batch size independent of k again, do you think it's worth testing batch sizes `max(2, k + random(-2,2))`?

Reply:
I'd say, yes for different batch sizes, but i am not so sure about what generator. I've tried to run the rest with batch sizes of 10*k and and 100*k and it does not seem to have much influence of the other tagged features.

One thing I wanted to tag before playing with the batch size was maybe the size of the final node's chain compared to k and maybe the size of the longest block application. This should give some ideas of what values could be interesting to generate for the batch size.